README.md

```diff
@@ -2,9 +2,8 @@
 
 # ChainRulesCore
 
-[![Travis](https://travis-ci.org/JuliaDiff/ChainRulesCore.jl.svg?branch=master)](https://travis-ci.org/JuliaDiff/ChainRulesCore.jl)
-[![Coveralls](https://coveralls.io/repos/github/JuliaDiff/ChainRulesCore.jl/badge.svg?branch=master)](https://coveralls.io/github/JuliaDiff/ChainRulesCore.jl?branch=master)
-[![PkgEval](https://juliaci.github.io/NanosoldierReports/pkgeval_badges/C/ChainRulesCore.svg)](https://juliaci.github.io/NanosoldierReports/pkgeval_badges/report.html)
```

Comment:
i added this one a while ago, so we could see if we pass on Julia Nightly version (without running it ourselves as part of the daily cron job), but AFAIK no one ever looks at it and it just means we occassionally have a "failing" badge on the README (which isn't great).

If anyone thinks we should keep the `PkgEval` let me know

Reply:
I personally like being able to tell at first first glance whether I can expect a package to work on Julia master, but I will admit that I am probably not that representative of the average user, so I don't really care.